<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Live Weather Checker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form { background-color: qlineargradient(spread:reflect, x1:0.502, y1:0.017, x2:0.513, y2:0.584955, stop:0 rgba(0, 138, 255, 255), stop:1 rgba(0, 255, 249, 255));}</string>
  </property>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button {
	font: 11pt &quot;Franklin Gothic Demi&quot;;
	background-color: rgb(255, 234, 0);
	border-radius: 10px;
}
QPushButton#button:hover{background-color: rgb(255, 200, 0);}
</string>
   </property>
   <property name="text">
    <string>Get Weather Forecast</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;
font-size:20px;</string>
   </property>
   <property name="text">
    <string>Enter a location name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#clearButton {
	font: 11pt &quot;Franklin Gothic Demi&quot;;
	background-color: rgb(255, 234, 0);
	border-radius: 10px;
}
QPushButton#clearButton:hover{background-color: rgb(255, 200, 0);}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="location">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="day_time">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="forecast">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="temperature">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="set_error">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="wind">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="humidity">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
